Formula/kumo.rb

```diff
@@ -0,0 +1,18 @@
+class Kumo < Formula
+  desc "Word Clouds in Java"
+  homepage "https://github.com/kennycason/kumo"
+  url "https://search.maven.org/remotecontent?filepath=com/kennycason/kumo-cli/1.13/kumo-cli-1.13.jar"
+  sha256 "c9ad525f7d6aec9e2c06cf10017e1e533f43b1b3c4df5aa0d4b137f8d563c5c6"
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "kumo-cli-1.13.jar"
+    bin.write_jar_script libexec/"kumo-cli-1.13.jar", "kumo"
+  end
+
+  test do
+    pipe_output("#{bin}/kumo --version", "Test Kumo version command")
```

Comment:
Why the use of `pipe_output()` here?

Reply:
No idea, copied from another example, there are ~250+ examples of it in codebase. I know that's not a good reason :)

Reply:
I found the source code here. https://github.com/Homebrew/brew/blob/e72fde418b439b45c6036774159b8e2479faaefe/Library/Homebrew/formula_assertions.rb

I believe the example I copied from also misused pipe_output. I will update to actually assert exit codes.

Reply:
I think you want something like:
```ruby
system bin/"kumo", "-i", "https://wikipedia.org", "-o", testpath/"wikipedia.png"
assert_predicate testpath/"wikipedia.png", :exist?, "Wordcloud was not generated!"
```

Reply:
Even better. Thanks. :) will update